packages/devtools_app/lib/src/network/network_controller.dart

```diff
@@ -351,11 +351,11 @@ class NetworkFilter {
       if (querySeparatorIndex != -1) {
         final value = part.substring(querySeparatorIndex + 1);
         if (value != '') {
```

Comment:
it could be nice to have an enum like
QueryFliter { final Iterable<string> keys; }
class so that you can reuse this parsing logic for all pages rather than writing it again on each page.